apis/v1alpha2/validation/httproute.go

```diff
@@ -64,7 +68,32 @@ func validateHTTPRouteUniqueFilters(rules []gatewayv1a2.HTTPRouteRule, path *fie
 			}
 		}
 
+		if errList := validateHttpRouteBackendRefs(rule.BackendRefs, path, i); len(errList) > 0 {
+			errs = append(errs, errList...)
+		}
 	}
 
 	return errs
 }
+
+func validateHttpRouteBackendRefs(ref []gatewayv1a2.HTTPBackendRef, path *field.Path, i int) field.ErrorList {
```

Comment:
A better name will be validateHTTPBackendUniqueFilters. Please add a godoc comment.

Reply:
9be59b9deca99e1af9fbdab5fc6a20a90bacd7dc